iati/tests/test_default.py

```diff
@@ -88,18 +93,36 @@ def test_default_codelist_invalid_at_all_versions(self, name, standard_version_o
         assert 'There is no default Codelist in version' in str(excinfo.value)
 
     def test_default_codelists_type(self, codelist_lengths_by_version):
```

Comment:
This test... does it have to type check? Surely it's more pythonic to check that it behaves, and has the expected attributes, that would be expected of an iati.codelist instead?

Reply:
https://github.com/IATI/pyIATI/pull/214/commits/d9a99c2c7589219fd09b8be8b75196543f865961

Probably doesn't, but can't immediately think of the best way to change this. Suggest leaving as-is, then modifying it along with a wider rework of Codelists.